src/client/styles/components/FilterPopup.scss

```diff
@@ -11,19 +11,6 @@
   a.popup-btn-open {
     padding-top: 3px;
   }
-
-  .popup-container {
-    visibility: hidden;
-    display: none;
-    opacity: 0;
-
-    &.active,
-    &:target {
```

Comment:
This is still needed for the no-js solution to close the popup. Make sure you test out the app's functionality before removing code that will break it.